scripts/validate-urls.py

```diff
@@ -0,0 +1,76 @@
+# This script extract http/https URLs from .md files and validate them.
```

Comment:
Can you add some usage instructions at the top, just to get people started. For example, where should I run this script and what are the required parameters (if any). An example of the basic usage would be great.

Reply:
I added more comments. PTAL. Thanks!